lib/src/widgets/yaru_info.dart

```diff
@@ -0,0 +1,118 @@
+import 'package:flutter/material.dart';
+import 'package:yaru/yaru.dart';
+
+enum YaruInfoType {
+  note,
+  tip,
+  important,
+  warning,
+  caution;
```

Comment:
And `error` here? Caution I would expect yellow for

Reply:
same as above, a red color could be used for other things as errors, WDYT?